openwisp_users/admin.py

```diff
@@ -194,7 +194,7 @@ class UserAdmin(MultitenantAdminMixin, BaseUserAdmin, BaseAdmin):
     add_form = UserCreationForm
     form = UserChangeForm
     ordering = ['-date_joined']
-    readonly_fields = ['last_login', 'date_joined']
+    readonly_fields = ['last_login', 'date_joined', 'password_updated']
```

Comment:
This is always shown but the field seems to be populated only when the feature is enabled.

Reply:
For some reason I can't see the `password_updated` field, can you?
Question: how difficult would it be to show this field below the main password field?

Reply:
![Screenshot from 2023-10-30 18-15-34](https://github.com/openwisp/openwisp-users/assets/32094433/16ca54d8-983a-4191-bcd0-10bbb691df41)